pybamm/__init__.py

```diff
@@ -51,6 +51,7 @@ def version(formatted=False):
 # Absolute path to the PyBaMM repo
 script_path = os.path.abspath(__file__)
 ABSOLUTE_PATH = os.path.join(os.path.split(script_path)[0], "..")
+PARAMETER_PATH = [os.getcwd(), os.path.join(os.path.split(script_path)[0], "input", "parameters")]
```

Comment:
note there is already a `pybamm.root_dir` command in `util.py`

Reply:
You'll need to reinstall pybamm to have the command installed, did you?

Reply:
Yes, but `pybamm_root_dir` yields the parent directory (in this case the `site-packages/`) which I think is a bit awkward..